### ISSUE TITLE
Color by progress status

### ISSUE THREAD

**Issue by tom2411** [axiel7/AniHyou-android#168]
**Request**
I wanted to know if it was possible to add a color code on the calendar and the all list to see if we are late in watching episodes or if we had seen all that is aired.
Thank you for your time.

Exemple 
To provide an exemple, you can see the app anitrend.
![Screenshot_2024-07-23-06-34-36-568_com.mxt.anitrend.jpg](https://github.com/user-attachments/assets/38ed52f3-78f7-45c5-8cad-4fd15d2202e7)

We have orange for anime airing with episode which I didn't watch yet.
Blue for anime airing but where i saw all episodes aired.
Green for anime which finished airing.

**Comment by axiel7** [comment]
> I would to have the badge with colors of similar scheme for the ones in my list. Currently its monochrome 

I agree with this, I'll try some colors and see how it looks with the score colors.

> Also have the highlighted status (when longed pressed to edit) to have the color changed to above scheme when selected

Yes.

> If possible have them customizable in the settings with a color picker

Not sure about this, I won't do it for now.

**Comment by axiel7** [comment]
Latest [beta ](https://github.com/axiel7/AniHyou-android/actions/runs/12073634596) version has the status indicator colored

![image](https://github.com/user-attachments/assets/83cb4c60-a2ac-4d0b-a289-2e5fde700ab7)
